src/ggrc/assets/javascripts/models/mappers/mappings-ggrc.js

```diff
@@ -496,6 +496,16 @@
         });
       }, 'CustomAttributeDefinition')
     },
+    // used by the Custom Roles admin panel tab
+    Roleable: {
+      custom_role_definitions: CustomFilter(
+        'WHAT_HERE???',  // TODO: or a different mapper helper? which?
```

Comment:
I've tried that initially, but got `can.Map: Object does not exist` error. Or is that actually some different kind of error?